src/FastSerialization/FastSerialization.cs

```diff
@@ -92,6 +106,7 @@ enum StreamLabelWidth
     sealed class SerializationConfiguration
     {
         public StreamLabelWidth StreamLabelWidth { get; set; }
+        public StreamReaderAlignment StreamReaderAlignment { get; set; } = StreamReaderAlignment.EightBytes;
```

Comment:
I interpreted SerializationConfiguration as describing what is supported in the file format. Other settings that solely affect implementation behavior/performance like the cache buffer size and whether the Stream should be left open are passed as parameters to the IOStreamStreamReader constructor. If we were following that convention I'd expect this StreamReaderAlignment to also be a constructor parameter rather than a property here.

I figured I'd mention it in case anyone cares, but I don't care much myself. If you and @brianrob are happy with this as a property it is fine for me.

Reply:
It's a good point.  The thing that I actually am thinking is that `SerializationConfiguration` gets used on both serialization and deserialization, and so I'm wondering if a deserialization-only property makes sense here.  I think I would prefer this as a parameter with a specified default value.  @josalem what do you think about just putting this as an optional parameter on the second constructor here?

Reply:
I can certainly move it. The first version I wrote up did it as a parameter. Thoughts on leaving it as an enum to restrict values?